server/routes/popVerifyRoutes.ts

```diff
@@ -11,118 +27,21 @@ export default function routes(): Router {
     router.post(routePath, asyncMiddleware(handler))
   router.use(setUpMultipartFormDataParsing())
 
-  get('/', async (req, res, next) => {
-    res.render('pages/pop-id/submit-photo')
-  })
+  get('/', renderSubmitPhoto)
+  get('/options', renderOptions)
+  post('/options', handleOptions)
+  get('/upload-photo', renderUploadPhoto)
+  post('/upload-photo', handleUploadPhoto)
```

Comment:
I was expecting this to be the only route that needs setUpMultipartFormDataParsing but making it more specific can be an improvement that we do later